src/index.html

```diff
@@ -21,7 +24,453 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="#"
+        class="img_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="img_header"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            src="../src/images/imac.jpeg"
+            alt="Display imac"
+            class="img_card"
+          />
+        </div>
+
+        <div class="card__bottom">
+          <h3 class="card__name-product">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__grade">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
```

Comment:
You need to use semantic tag for text, fix it everywhere
```suggestion
            <p class="card__reviews">Reviews: 5</p>
```